cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/bridge_hub_rococo_config.rs

```diff
@@ -51,12 +52,37 @@ parameter_types! {
 	pub BridgeHubRococoUniversalLocation: InteriorMultiLocation = X2(GlobalConsensus(Rococo), Parachain(ParachainInfo::parachain_id().into()));
 	pub WococoGlobalConsensusNetwork: NetworkId = NetworkId::Wococo;
 	pub ActiveOutboundLanesToBridgeHubWococo: &'static [bp_messages::LaneId] = &[DEFAULT_XCM_LANE_TO_BRIDGE_HUB_WOCOCO];
-	pub PriorityBoostPerMessage: u64 = 921_900_294;
+	// see the `FEE_BOOST_PER_MESSAGE` constant to get the meaning of this value
+	pub PriorityBoostPerMessage: u64 = 182_044_444_444_444;
+
+	pub AssetHubRococoParaId: cumulus_primitives_core::ParaId = bp_asset_hub_rococo::ASSET_HUB_ROCOCO_PARACHAIN_ID.into();
 
 	pub FromAssetHubRococoToAssetHubWococoRoute: SenderAndLane = SenderAndLane::new(
-		ParentThen(X1(Parachain(1000))).into(),
+		ParentThen(X1(Parachain(AssetHubRococoParaId::get().into()))).into(),
 		DEFAULT_XCM_LANE_TO_BRIDGE_HUB_WOCOCO,
 	);
+
+	pub CongestedMessage: Xcm<()> = sp_std::vec![Transact {
+		origin_kind: OriginKind::Xcm,
+		require_weight_at_most: bp_asset_hub_rococo::XcmBridgeHubRouterTransactCallMaxWeight::get(),
+		call: bp_asset_hub_rococo::Call::ToWococoXcmRouter(
+			bp_asset_hub_rococo::XcmBridgeHubRouterCall::report_bridge_status {
+				bridge_id: Default::default(),
+				is_congested: true,
+			}
+		).encode().into(),
+	}].into();
+
+	pub UncongestedMessage: Xcm<()> = sp_std::vec![Transact {
+		origin_kind: OriginKind::Xcm,
+		require_weight_at_most: bp_asset_hub_rococo::XcmBridgeHubRouterTransactCallMaxWeight::get(),
+		call: bp_asset_hub_rococo::Call::ToWococoXcmRouter(
+			bp_asset_hub_rococo::XcmBridgeHubRouterCall::report_bridge_status {
+				bridge_id: Default::default(),
+				is_congested: false,
+			}
+		).encode().into(),
+	}].into();
```

Comment:
@serban300 I dont know, is it really worth to introduce new function for that? Is there a way how avoid `parameter_types` definitions with that function?

I think it could be ok, if we could just use it like:
```
type CongestedMessage = congested_message_builder(true);
type UncongestedMessage = congested_message_builder(false);
```

Reply:
I don't think we can avoid `parameter_types`. The function wound need to be `const` and I don't think we can do that.

But even so, having a function would deduplicate this code.